firebase.js

```diff
@@ -0,0 +1,9 @@
+const admin = require('firebase-admin');
+
+require('dotenv').config();
+
+const credentials = require('./auth-dev-90d20-firebase-adminsdk-7k0yw-5c633a4fb2.json');
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './auth-dev-90d20-firebase-adminsdk-7k0yw-5c633a4fb2.json'.